mail_activity_reminder/models/mail_activity.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from pytz import timezone, UTC
+
+from odoo import _, api, fields, models
+
+
+class MailActivity(models.Model):
+    _inherit = 'mail.activity'
+
+    last_reminder = fields.Date()
+
+    @api.model
+    def _get_activities_to_remind(self):
+        utc_now = fields.Datetime.now().replace(tzinfo=UTC)
+
+        def _should_remind(activity):
```

Comment:
don't hide this inside the method: move it to an instance method (then don't pass `activity` and use `self)

Reply:
also true